sql/database.sql

```diff
@@ -5046,21 +5041,6 @@ CREATE TABLE `patient_data` (
   `squad` varchar(32) NOT NULL default '',
   `fitness` int(11) NOT NULL default '0',
   `referral_source` varchar(30) NOT NULL default '',
-  `usertext1` varchar(255) NOT NULL DEFAULT '',
-  `usertext2` varchar(255) NOT NULL DEFAULT '',
-  `usertext3` varchar(255) NOT NULL DEFAULT '',
-  `usertext4` varchar(255) NOT NULL DEFAULT '',
-  `usertext5` varchar(255) NOT NULL DEFAULT '',
-  `usertext6` varchar(255) NOT NULL DEFAULT '',
-  `usertext7` varchar(255) NOT NULL DEFAULT '',
-  `usertext8` varchar(255) NOT NULL DEFAULT '',
-  `userlist1` varchar(255) NOT NULL DEFAULT '',
-  `userlist2` varchar(255) NOT NULL DEFAULT '',
-  `userlist3` varchar(255) NOT NULL DEFAULT '',
-  `userlist4` varchar(255) NOT NULL DEFAULT '',
-  `userlist5` varchar(255) NOT NULL DEFAULT '',
-  `userlist6` varchar(255) NOT NULL DEFAULT '',
-  `userlist7` varchar(255) NOT NULL DEFAULT '',
```

Comment:
removing user text and user list will break patient portal

Reply:
If the portal uses it, I will want to find out where/why.  If it needs fields, I will add them.  No undefined elements should be in the patient_data table, and since this table is user-configurable, things that look at it should not expect to use hard-coded demands for a field to exist, or expect any sequence of results from an unordered array.

Reply:
Heavy sigh...
Otay.
But it should not be REQUIRED anywhere.  External ID's should be a vert list.

Reply:
I agree with @aethelwulffe on this, no apps should rely on fields that the user can change and reuse.

Reply:
They can change and reuse any field in the layout form, not just these

Reply:
Understood.  Layouts is just a bit too powerful in that regard.  Should set some minimal restrictions.  That has been a problem in the past (good old billing note for one) and has resulted in many broken systems.